app/actions/balancesActions.js

```diff
@@ -37,11 +37,11 @@ async function getBalances ({ net, address, tokens }: Props) {
 
   // asset balances
   promises.push((async () => {
-    const assetBalances = await api.neonDB.getBalance(net, address)
+    const assetBalances = await api.loadBalance(api.getBalanceFrom, { net, address })
 
     return {
-      [ASSETS.NEO]: assetBalances.assets.NEO.balance.toString(),
-      [ASSETS.GAS]: assetBalances.assets.GAS.balance.round(COIN_DECIMAL_LENGTH).toString()
+      [ASSETS.NEO]: assetBalances.balance.assets.NEO.balance.toString(),
+      [ASSETS.GAS]: assetBalances.balance.assets.GAS.balance.round(COIN_DECIMAL_LENGTH).toString()
```

Comment:
Does the shape of this endpoint differ between neoscan and neonDB?  Does one have a `balance` key and the other doesn't?

Reply:
I can take a look again but I am confused as to why yours worked before. Both endpoints should return this same format. Will take another look

Reply:
You know, actually it might have the data under both `assets` and `balance.assets`.  I remember seeing duplicate data structure coming back from neonDB when I added this.

Reply:
The duplicate data structure was when i was transiting to the new model and retained the old models for a while before deprecation. The old code looks correct though.

Reply:
Thanks @snowypowers :)